### PR TITLE
Started Notebook for WFM Cloud Water Data

### PR DESCRIPTION
**Pull request by AMDeitsch**

### CODE REVIEW

File: notebooks/example-workflows/wfm-cloud-water.ipynb
Comment (cyschneck) [inline notebooks/example-workflows/wfm-cloud-water.ipynb:1]: Headers still includes the default Pythia information for including a relevant image

**Review by cyschneck — COMMENTED** [review]
Is this notebook just a draft to have included on the main page?

**Comment by AMDeitsch** [comment]
> Is this notebook just a draft to have included on the main page?

I suppose a pull request wasn't necessary until it's finished, yeah. 🤷‍♂️

**Review by AMDeitsch — COMMENTED** [review]
Will continue to develop this. No need to merge yet.

**Pull request closed without merging**